src/components/Schedule/AvailabilityBar.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FC } from "react";
+import { ProgressBar } from "./ProgressBar";
+
+interface IProps {
+  currentReservation: number;
+  maxReservation: number;
+}
+
+export const AvailabilityBar: FC<IProps> = ({
+  currentReservation,
+  maxReservation
+}) => {
+  return (
+    <>
+      <div className="flex justify-between lg:pt-20 pt-8 text-12 font-lato font-bold">
+        <div>Obsadenosť</div>
+        <div className="flex content-end">
+          <div>{currentReservation}/</div>
+          <div>{maxReservation}</div>
+        </div>
+      </div>
+      <div className="pt-8 border-b border-solid border-lilac relative flex items-center">
```

Comment:
![image](https://user-images.githubusercontent.com/43435775/58557009-9e1eec80-821d-11e9-9fb7-833cd141c987.png) something strange happens here.. in storybook, progress bar black line on the first 'Reservation' has correct position (starts about 1px above the grey line and ends about 1px under the grey line), but on the second Reservation component starts a little bit upper and ends on the grey line.  But this is the same component right? So, it's just a bug of storybook or how is that possible? :D  Mobile version of progress bar is on both Reservation components in incorrect position